jamal-snippet/src/main/java/javax0/jamal/snippet/Sort.java

```diff
@@ -0,0 +1,134 @@
+package javax0.jamal.snippet;
+
+import javax0.jamal.api.*;
+import javax0.jamal.tools.InputHandler;
+import javax0.jamal.tools.Params;
+import javax0.jamal.tools.Scan;
+
+import java.text.Collator;
+import java.util.*;
+import java.util.function.Function;
+import java.util.regex.Pattern;
+import java.util.stream.Stream;
+
+import static java.util.stream.Collectors.toList;
+import static javax0.jamal.tools.InputHandler.skipWhiteSpaces;
+import static javax0.jamal.tools.Params.holder;
+
+public class Sort implements Macro, InnerScopeDependent {
+
+	@Override
+	public String evaluate(Input in, Processor processor) throws BadSyntax {
+		final var separator = holder("separator").orElse("\n").asPattern();
+		final var join = holder("join").orElse("\n").asString();
+		final var locale = holder("locale").asString();
```

Comment:
I would also add alias `collatingOrder` or something like that. This is not the locale, this is the sorting order.